### ISSUE TITLE
Compose thread before sending

### ISSUE THREAD

**Issue by forestsystem** [elk-zone/elk#1073]
### Clear and concise description of the problem

On Twitter, there's a nice feature: after you write a tweet, you can click a button to add another tweet under that one, so you can write a fairly long thread (I think 25ish tweets?) before sending it. I used this feature a lot and liked it very much.

On regular Mastodon, you have to send your first toot before you can start drafting the next one. Because I like to finish writing threads before I post them, I often use a workaround of composing my threads in my computer's notes app, then pasting individual toots in one at a time. This is kind of annoying to have to do. :(

I think it would be very cool if Elk supported draft threads like on Twitter!

### Suggested solution

Add a little "add toot" button when composing a toot, then allow some number of subsequent toots to be drafted before posting.

**Comment by edimitchel** [comment]
The question is: is Thread still needed as we have now higher toot capacity?

That's said, it could be very useful to let build multiple toots in once

**Comment by forestsystem** [comment]
I find threads very useful. I'm pretty long-winded, haha - I just posted one with 12 toots. Maybe I should move to a different instance or something, but... I actually really like how threads break up thoughts into easy-to-digest bites. It helps me process things better.

**Comment by edimitchel** [comment]
Fully understand that, thread editor needs to land into Elk for sure!

**Comment by ZMYaro** [comment]
I agree, even though some instances support longer posts, not all do, and sometimes it makes sense to break a thought into multiple posts anyway.

**Comment by snail-coupe** [comment]
I was just abut to suggest this too.  An alternative option, which could be nice, is to let you compose i one long go but, if it's over the limit, automatically offer to split it into chunks for you, with some sensible "logic" and handling when a user writes a 600 character long paragraph.  It could even add the 1/x for you.

**Comment by ayo-run** [comment]
Hi all! We now have this capability to "add item" to the thread before publishing them all. Please enjoy, and see if some behavior is missing. Thanks! https://main.elk.zone

<img width="660" height="464" alt="Image" src="https://github.com/user-attachments/assets/29e22017-6031-4fbc-983c-2b17d0609f45" />
